### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by choppa98**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -329,82 +358,173 @@ def output_getter(k, v):
...
+regions_to_aggregate = aggregate_regions.loc[0,:].values.flatten().tolist()[1].split(',')
+agg_region = aggregate_regions.loc[0,:].values.flatten().tolist()[0]
+regions_to_aggregate = [x.strip() for x in regions_to_aggregate]
+
+regions_to_aggregate1 = aggregate_regions.loc[1,:].values.flatten().tolist()[1].split(',')
+agg_region1 = aggregate_regions.loc[1,:].values.flatten().tolist()[0]
+regions_to_aggregate1 = [x.strip() for x in regions_to_aggregate1]
+
+regions_to_aggregate2 = aggregate_regions.loc[2,:].values.flatten().tolist()[1].split(',')
+agg_region2 = aggregate_regions.loc[2,:].values.flatten().tolist()[0]
```
Comment (LippkauFelix) [inline main.py:380]: same here

**Review by LippkauFelix — CHANGES_REQUESTED** [review]

File: main.py [hunk 2]
```diff
@@ -329,82 +358,173 @@ def output_getter(k, v):
...
-
-    ## Remove code duplicates as well.
-    ks = ts.groupby(["code", "Region", "Variable"], as_index=False).agg("sum", numeric_only=True)
-    ts = pd.merge(
-        ks, ts[[i for i in ts.columns if not str(i).isnumeric()]], how="inner", on=["code", "Region", "Variable"]
-    ).copy()
-    ts = ts.drop_duplicates()
-    aggreagte_dfs.append(ts)
-out = pd.concat([out.copy()] + aggreagte_dfs)
+out = out[out.Region != "DEU"]
```
Comment (LippkauFelix) [inline main.py:361]: at this stage of code out should not include any aggregated region? Please check this again.

**Review by LippkauFelix — CHANGES_REQUESTED** [review]

File: main.py (hunk 2 above)
Comment (choppa98) [reply main.py:361]: it does have deu, eu and panEU

File: main.py (hunk 1 above)
Comment (choppa98) [reply main.py:380]: same [0] will give agg_region in that row and [1] all the regions to aggregate

File: main.py [hunk 3]
```diff
@@ -329,82 +358,173 @@ def output_getter(k, v):
...
+out = out[['code','Model', 'Scenario', 'Region', 'Variable', 'Unit', '2010', '2015', '2020', '2025','2030', '2035', '2040', '2045', '2050']]
+
+df = out.copy()
+#print(out)
+regions_to_aggregate = aggregate_regions.loc[0,:].values.flatten().tolist()[1].split(',')
+agg_region = aggregate_regions.loc[0,:].values.flatten().tolist()[0]
+regions_to_aggregate = [x.strip() for x in regions_to_aggregate]
+
+regions_to_aggregate1 = aggregate_regions.loc[1,:].values.flatten().tolist()[1].split(',')
+agg_region1 = aggregate_regions.loc[1,:].values.flatten().tolist()[0]
```
[the remark this answers is not in the document]
Comment (LippkauFelix) [reply main.py:376]: sorry, you are right! Unfortunately we are at the starting point for the error that Alex mentioned

[the remark this answers is not in the document]
Comment (choppa98) [reply main.py:376]: ok I totally forgot that we didn't have 'scenario' until the end in main.py, so groupby scenario returns an empty dataframe. I will push the changes.

**Review by LippkauFelix — APPROVED** [review]

**Review by AlexanderBurkhardt-IER — APPROVED** [review]

**Review by AlexanderBurkhardt-IER — COMMENTED** [review]
Works

**Pull request merged**